x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_landing.scss

```
@@ -0,0 +1,15 @@
+.crawlerLanding {
+  &__panel {
+    overflow: hidden;
+    background-image: url('./assets/bg_crawler_landing.png');
+    background-size: 45%;
+    background-repeat: no-repeat;
+    background-position: right -2rem;
+
+  }
+
+  &__wrapper {
+      max-width: 50rem;
+    }
+  }
```

It looks like this is causing the CI failures: https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/128437/execution/node/179/log/

```suggestion
  }

  &__wrapper {
    max-width: 50rem;
  }
```